src/assisted_test_infra/test_infra/helper_classes/nodes.py

```diff
@@ -245,7 +245,7 @@ def _all_nodes_allow_ssh_connection():
 
         waiting.wait(
             lambda: _all_nodes_allow_ssh_connection(),
-            timeout_seconds=180,
+            timeout_seconds=10 * consts.MINUTE,
```

Comment:
10 minutes sounds a lot to wait for nodes to have the ssh connection. How did we got to this number?

Reply:
just noticed its a draft. ignore me

Reply:
Yeah, 3 mins seems to low according to failed tests, we can try also with 5 mins.

Reply:
Note that on QE CI we run tests in parallel of 3 (meaning possibly 15+ nodes at a time), and we dont have ssh failures. We do have rare cases where some nodes fails to boot but mostly stable. So i would also suggest to check other possible reasons for your failures